modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-enlargement.adoc
+
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+A virtual machine (VM) disk has a default or initial storage capacity when you create it. However, you can enlarge the disk by resizing the disk's persistent volume claim (PVC).
+
+VM disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
```

Comment:
I would provide the links suggested by Adam describing how to consume the resized storage capacity:
https://docs.microsoft.com/en-us/windows-server/storage/disk-management/extend-a-basic-volume
https://access.redhat.com/solutions/29095
https://access.redhat.com/solutions/24770

Reply:
LGTM